src/cli/test_command.zig

```diff
@@ -1371,8 +1215,14 @@ pub const TestCommand = struct {
                 break :brk scanner.fs.top_level_dir;
             };
 
-            scanner.scan(dir_to_scan);
-            scanner.dirs_to_scan.deinit();
+            scanner.scan(dir_to_scan) catch |err| switch (err) {
+                error.OutOfMemory => bun.outOfMemory(),
+                error.DoesNotExist => {
+                    Output.prettyErrorln("<red>Failed to scan non-existent root directory for tests:<r> {s}", .{dir_to_scan});
```

Comment:
Can you paste a screenshot of where this happens? having trouble making this one happen

Reply:
<img width="974" alt="image" src="https://github.com/user-attachments/assets/1ad959f2-2972-48bf-b80c-8c4a06c03a57" />